src/scripts/docker-buildx-cleanup.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+# to prevent filesystem corruption, clean up multi-arch binary format handlers from the host prior to
+# saving updated cache
+docker run --privileged --rm tonistiigi/binfmt --uninstall qemu-*
```

Comment:
One concern I have is that it appears we only create the `buildx` context and run the `binfmt` install if we're using a private registry (the contents of the else block [here](https://github.com/CircleCI-Public/aws-ecr-orb/blob/14861880e15d2323048d623370c33fb42c5c82a8/src/scripts/docker-buildx.sh#L66)), but in this case we'd be assuming that the install had happened. Do we intend for the context creation / `binfmt` install to be guarded by the use of a private repo? Does the `binfmt` install need to be coupled to the context creation?